### PR TITLE
UI revamp of Gyaan Karmayogi

### PR DESCRIPTION
**Pull request by venkykandagaddala**

### CODE REVIEW

File: project/ws/app/src/lib/routes/knowledge-resource/components/knowledge-card/knowledge-card.component.html [hunk 1]
```diff
@@ -1,57 +1,45 @@
...
-            {{ resource?.totalFiles?.doc }}
-        </p>
-    </span>
+        <!-- <span class="attach_links margin-right-m" *ngIf="resource.totalFiles.pdf || resource.totalFiles.pdf >0">
+            <p class="mat-body-2" i18n>  <img src="assets/icons/pdf.svg" class="pdfIcon margin-right-xs">
+                {{ resource?.totalFiles?.pdf }}
+            </p>
+        </span> -->
+
+        <span class="attach_links" *ngIf="resource.additionalProperties.krFiles && resource.additionalProperties.krFiles.length > 0">
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/knowledge-resource/components/knowledge-card/knowledge-card.component.html]: Please put optional checks

File: project/ws/app/src/lib/routes/knowledge-resource/components/knowledge-card/knowledge-card.component.html [hunk 2]
```diff
@@ -1,57 +1,45 @@
...
+            <p class="mat-body-2" i18n>
+                <span *ngFor="let item of resource.additionalProperties.krFiles | slice:0:1; index as i">
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/knowledge-resource/components/knowledge-card/knowledge-card.component.html]: Please put optional check

File: project/ws/app/src/lib/routes/knowledge-resource/components/knowledge-card/knowledge-card.component.html (hunk 1 above)
Comment (venkykandagaddala) [reply project/ws/app/src/lib/routes/knowledge-resource/components/knowledge-card/knowledge-card.component.html]: updated

File: project/ws/app/src/lib/routes/knowledge-resource/components/knowledge-card/knowledge-card.component.html (hunk 2 above)
Comment (venkykandagaddala) [reply project/ws/app/src/lib/routes/knowledge-resource/components/knowledge-card/knowledge-card.component.html]: updated

**Pull request merged**